### PR TITLE
Add dependabot on release branch

### PR DESCRIPTION
**Pull request by tomaskikutis**

### CODE REVIEW

**Comment by petrjasek** [comment]
it only works on 1 branch per repo, so with that change it would stop working on develop

**Comment by tomaskikutis** [comment]
https://github.com/dependabot/dependabot-core/issues/388

Are you sure? I read about people being able to configure it. Also it appears the feature was merged in 2018

**Comment by petrjasek** [comment]
we can test this, don't remember now if it wasn't working for security updates or version updates

File: client/index.ts [hunk 1]
```diff
@@ -3,10 +3,6 @@ import {startApp} from 'superdesk-core/scripts/index';
 setTimeout(() => {
     startApp(
         [
-            {
-                id: 'broadcasting',
-                load: () => import('superdesk-core/scripts/extensions/broadcasting'),
```
Comment (petrjasek) [inline client/index.ts:8]: is it broken?

Comment (tomaskikutis) [reply client/index.ts:8]: it's duplicated, the same thing is imported on line 23, except with custom configuration so I chose to keep that

**Pull request merged**